specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-02-01-preview/scanningService.json

```diff
@@ -0,0 +1,7565 @@
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "dataSourceName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "scanName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "runId",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "scanLevel",
+            "type": "string",
+            "enum": [
+              "Full",
+              "Incremental"
+            ],
+            "x-ms-enum": {
+              "name": "ScanLevelType",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OperationResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ScanResult_RunScan": {
+            "$ref": "./examples/ScanResult_RunScan.json"
+          }
+        }
+      }
+    },
+    "/datasources/{dataSourceName}/scans/{scanName}/runs/{runId}/:cancel": {
+      "post": {
+        "tags": [
+          "ScanResult"
+        ],
+        "description": "Cancels a scan",
+        "operationId": "ScanResult_CancelScan",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "dataSourceName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "scanName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "runId",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OperationResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ScanResult_CancelScan": {
+            "$ref": "./examples/ScanResult_CancelScan.json"
+          }
```

Comment:
Operations with a 202 response should specify `x-ms-long-running-operation: true`.